server/channels/utils/imgutils/gif_test.go

```diff
@@ -86,4 +86,13 @@ func TestCountGIFFrames(t *testing.T) {
 
 		assert.Error(t, err)
 	})
+
+	t.Run("should return an error for excessively large compressed data", func(t *testing.T) {
+		b, err := readTestFile(t, "large_lzw_frame.gif")
+		require.NoError(t, err)
+
+		_, err = CountGIFFrames(bytes.NewReader(b))
+
+		assert.Error(t, err)
```

Comment:
Suggest adding an extra line `require.Equal(t, errTooMuch, err)`. I was a bit confused because the actual error is returned from `lzwr.Read` and not the `io.Copy` itself.

Reply:
Done.